.vagrant.mk

```diff
@@ -14,15 +14,15 @@
 
 .PHONY: vagrant-start
 vagrant-start:
-	@cd scripts/vagrant; vagrant up
+	@cd scripts/vagrant; vagrant up --no-parallel; source env.sh
```

Comment:
the source env.sh here will have no effect, as it dies with the shell spawned by make.

Reply:
You're right!
The idea was to export the KUBECONFIG env so that parent shell knows about it. Without that, subsequent kubectl calls will fail. It prevent a fresh install to work out of the box with 'make k8s-deploy' after a 'git clone' of the project.